doc/developers.rst

```diff
@@ -99,6 +99,7 @@ unofficial Ubuntu repository, which has our software packages available to downl
                         python3-dev \
                         qtbase5-dev \
                         libqt5svg5-dev \
+                        libxcb-xfixes0-dev \
```

Comment:
Without this package we can't finish `make install` for qt5 library with success

Reply:
I've never even heard of `libxcb-xfixes0-dev`, and it's certainly not anything we explicitly install in either our CI builds or the official PPA builds.

Hmm. In [our recent PPA builds](https://launchpadlibrarian.net/583137755/buildlog.txt.gz) I can see where `libxcb-xfixes0` itself is pulled in, along with the rest of the xcb libs, but the `-dev` packages are never pulled in and the build runs fine.

What build failures were you getting without `libxcb-xfixes0-dev`? Can you post the output from `cmake`/`g++`/`clang++`?

Reply:
Oh, sorry, you said `make install`... So the build is successful, but the _install_ fails without `libxcb-xfixes0-dev`? I'm even more confused, if that's the case.

Reply:
@ferdnyc Yes. My patch is useful for developers who would like to replicate `make install` case from the documentation :-) Is it something, that we can merge?